### PR TITLE
Change snapcraft to build from github

### PR DESCRIPTION
**Pull request by ckti**
Change snapcraft to build from github, change contact from Instagram to Telegram

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -45,14 +45,13 @@ description: |
...
+  Get source code from Github: https://github.com/ioncoinicore/ion.git
   
   Chat with Support on Discord: https://discord.gg/vuZn7gC
   
   Contact us on social networks and forums:
   
-  - Instagram: https://t.me/ionomy
+  - Telegram:  https://t.me/ionomy
   - Twitter:   https://twitter.com/ionomics
   - Reddit:    https://www.reddit.com/r/ionomy/
```
Comment (cryptolinux) [inline snap/snapcraft.yaml]: @korvasc should first confirm contact us on social networks and forums has valid links, We do not have access to all those links therefore those should be removed or updated

Comment (ckti) [reply snap/snapcraft.yaml]: have removed all but ioncoincore links

Comment (ckti) [reply snap/snapcraft.yaml]: have removed applicable links

**Review by ioncoincore — APPROVED** [review]
ACK

**Pull request merged**